package.json

```diff
@@ -21,11 +22,14 @@
   },
   "devDependencies": {
     "bluebird": "^2.9.12",
+    "chai": "^2.2.0",
     "loopback-datasource-juggler": "^2.15.0",
+    "memwatch": "^0.2.2",
     "mocha": "^2.1.0",
     "rc": "^1.0.0",
     "semver": "^4.2.0",
-    "should": "^5.0.0"
+    "should": "^5.0.0",
+    "sinon": "^1.14.1"
```

Comment:
Since `sinon` is used in one place only, please consider rewriting that place in such way that we are not adding another dev dependency.